_projects/gufe.md

```diff
@@ -0,0 +1,10 @@
+---
+name: gufe
```

Comment:
@richardjgowers do we want to advertise gufe in this way? I was under the impression that it technically is an openfe component "for those in the know"

Reply:
Ehhh not sure, I guess as long as it's not at the top of the list (see comment below)

Reply:
I was planning on documenting gufe as part of the openfe package, it's just split out for logistical reasons, but maybe we should dogfood it more than that

Reply:
This is intended to be a "for the moment" list. In the future, we might remove gufe. Right now, my goal is for the projects page to basically say "see, we really are doing a lot of work!" to potential funders -- listing many projects makes us look a little more thoroughly established. (We might add the PL benchmarks repo here, too.) I'm thinking of it like a CV: when you're young and haven't done much, you put everything on there. Once you have a decent bit of experience, you only include the highlights.